extensions/amp-form/0.1/amp-form.js

```diff
@@ -291,7 +308,12 @@ export class AmpForm {
       xhrUrl = addParamsToUrl(url, values);
     } else {
       xhrUrl = url;
-      body = createFormDataWrapper(this.win_, this.form_);
+      if (this.encType_ === 'application/x-www-form-urlencoded') {
+        body = new URLSearchParams(this.getFormAsObject_());
```

Comment:
As discussed over Chat, I don't think we can use `URLSearchParams` within the Runtime.

For example, within `src/xhr-utils.js`, it says:
```
 ...
 * algorithm. Other serialization-needing types such as `URLSearchParams`
 * (which is not supported in IE and Safari) and `FederatedCredentials` are
 * not used in AMP runtime.
```

Reply:
After a quick glance, would `src/url.serializeQueryString` perform the same function?

Reply:
Thanks for the recommendation on using url.serializeQueryString, this does perform the same function.  I have replaced with this function.